doc/content/getting-started/cli/_index.md

```diff
@@ -10,4 +10,6 @@ Although the web interface of {{% tts %}} (the Console) currently has support fo
 
 You can use the CLI on your local machine and on the server.
 
+For a complete list of CLI commands, see the [CLI reference]({{< ref "reference/cli" >}}).
```

Comment:
Maybe add a link like this to troubleshooting as well, as a user might not expect to find it in the reference